### PR TITLE
Make CI a subtree

### PR DESCRIPTION
**Pull request by 8W9aG**
* Removes user confusion when pulling in submodules

### CODE REVIEW

**Comment by 8W9aG** [comment]
@spotify/objc-dev

**Comment by rastersize** [comment]
👍

**Comment by rastersize** [comment]
@8W9aG Should we also switch over to Codecov in the same go?

**Comment by rastersize** [comment]
Oh and you’ll need to define at least two more environment variables:

- `EXPECTED_LICENSE_HEADER_FILE `
- `LICENSED_SOURCE_FILES_GLOB `

See how it’s done in https://github.com/spotify/SPTPersistentCache/blob/master/.travis.yml#L51-L52 and the docs in [ios-ci](https://github.com/spotify/ios-ci#travis-ci).

**Comment by 8W9aG** [comment]
@rastersize indeed we should

File: other/expected_license_header.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2016 Spotify AB.
```
Comment (rastersize) [inline other/expected_license_header.txt]: I think the SPTDataLoader license header has `2015-2016` as the year.

**Comment by rastersize** [comment]
You’ll want to change the coverage badge in the readme as well:
```diff
- [![Coverage Status](https://coveralls.io/repos/spotify/SPTDataLoader/badge.svg?branch=master&service=github)](https://coveralls.io/github/spotify/SPTDataLoader?branch=master)
+ [![Coverage Status](https://codecov.io/github/spotify/SPTDataLoader/coverage.svg?branch=master)](https://codecov.io/github/spotify/SPTDataLoader?branch=master)
```

File: other/expected_license_header.txt (hunk 1 above)
Comment (8W9aG) [inline other/expected_license_header.txt]: 👍  Will make the change

**Comment by 8W9aG** [comment]
Requires https://github.com/spotify/ios-ci/pull/13 and a subtree bump before merging

**Comment by rastersize** [comment]
👍

**Pull request merged**